playground/ci/grpc_client.py

```diff
@@ -0,0 +1,93 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import grpc
+
+from api import api_pb2_grpc, api_pb2
+from config import SERVER_ADDRESS
+
+
+class GRPCClient:
+    """GRPCClient is gRPC client for sending a request to the backend."""
+
+    def __init__(self):
+        self._channel = grpc.aio.insecure_channel(SERVER_ADDRESS)
+        self._stub = api_pb2_grpc.PlaygroundServiceStub(self._channel)
+
+    async def run_code(self, code: str, sdk: api_pb2.Sdk) -> str:
+        """Run example by his code and SDK
+
+            Args:
+                code: code of the example.
+                sdk: SDK of the example.
+
+            Returns:
+                pipeline_uuid: uuid of the pipeline
+            """
+        request = api_pb2.RunCodeRequest(code=code, sdk=sdk)
+        response = await self._stub.RunCode(request)
+        return response.pipeline_uuid
```

Comment:
Each method (`RunCode`/`CheckStatus`/...) could return error (for example if for `CheckStatus` pipelineId is incorrect). I think we need to catch these cases too.